src/test/tests/faulttolerance/badfile.py

```diff
@@ -19,15 +19,15 @@
 f.close()
 OpenDatabase("junk.silo")
 e = GetLastError()
-TestText("badfile_01", e)
+#TestText("badfile_01", e)
 
 # Make zero-length VTK file.  The VTK reader crashes in this case.  If the
 # VTK reader is ever improved, then we'll have to locate a reader that
 # crashes successfully.  (Maybe we'll write one.)
 f = open("junk.vtk", "w")
 f.close()
 OpenDatabase("junk.vtk")
-e = GetLastError()
-TestText("badfile_02", e)
+#e = GetLastError()
+#TestText("badfile_02", e)
```

Comment:
Should these be commentd out

Reply:
- [ ] check into this

good catch, thanks!